code/__HELPERS/duplicating.dm

```diff
@@ -0,0 +1,83 @@
+///List of all vars that will not be copied over when using duplicate_object()
+GLOBAL_LIST_INIT(duplicate_forbidden_vars, list(
+	"AIStatus",
+	"actions",
+	"appearance",
+	"area",
+	"ckey",
```

Comment:
I see some of the old ones like stored_end missing, intended?

Reply:
Readded + some extra